packages/strapi-design-system/src/Table/Cell.js

```diff
@@ -0,0 +1,47 @@
+import React, { useEffect, useRef } from 'react';
+import PropTypes from 'prop-types';
+
+const useCell = (isFocused) => {
+  const mountedRef = useRef(false);
+  const cellRef = useRef(null);
+
+  useEffect(() => {
+    if (mountedRef.current && isFocused) {
+      cellRef.current.focus();
+    }
+  }, [isFocused]);
+
+  useEffect(() => {
+    mountedRef.current = true;
```

Comment:
Why don't we need the cleanup here?

Reply:
not sure why you'd like to cleanup?

Reply:
Settings the `mountedRef.current`  to false

Reply:
This still not answers "why" but how 😛

Reply:
Making sure the effect above is not ran when the components unmounts. Usually I take the other approach:


```
const mountedRef = useRef(true);

useEffect(() => {
  // ...
}, [isFocused)

useEffect(() => {
 return () => (mountedRef.current = false)
}, [])
```

Reply:
The goal of this is to make sure the .focus is not called on mount. Verifying the value when unmounting does not seem necessary to me